blueprints/component-test/qunit-files/tests/__testType__/__path__/__test__.js

```diff
@@ -8,7 +8,7 @@ moduleForComponent('<%= componentPathName %>', '<%= friendlyTestDescription %>',
 });
 
 test('it renders', function(assert) {
-  <% if (testType === 'integration' ) { %>// Set any properties with this.set('myProperty', 'value');
+<% if (testType === 'integration' ) { %>// Set any properties with this.set('myProperty', 'value');
```

Comment:
I see the issue you're trying to solve, but this check will need to go to the line above to get rid of the extra line.